src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -301,7 +304,12 @@ const Contact: React.FC<Props> = ({
                             <Typography variant="subtitle1">{name}</Typography>
                           </Link>
                           <Typography variant="subtitle2">
-                            {`Current: ${statusTitle || ''} ${
+                            {`Current: ${
+                              getLocalizedContactStatus(
+                                t,
+                                status as StatusEnum,
+                              ) || ''
+                            } ${
```

Comment:
It returns `''` is it doesn't match any status
```suggestion
                            {`Current: ${
                              getLocalizedContactStatus(
                                t,
                                status as StatusEnum,
                              )
                            } ${
```